app/api/Datasets.scala

```diff
@@ -2408,6 +2359,122 @@ class  Datasets @Inject()(
+      case Some(u: User) => {
+        creatorName = u.fullName
+        creatorOrcid = u.profile match {
+          case Some(p: Profile) => p.orcidID.getOrElse("")
+          case None => ""
+        }
+      }
+      case None => {}
+    }
+
+    // ---------- REQUIRED FIELDS ----------
+    // Identifier (DOI)
+    // TODO: We don't have a DOI yet in most cases. It seems that DataCite v4+ requires one, however RDA does not.
+    s += "<identifier identifierType=\"DOI\">:none</identifier>\n"
+
+    // Creators
+    s += "<creators>\n"
+    s += "\t<creator>\n"
+    s += "\t\t<creatorName>"+creatorName+"</creatorName>\n"
+    if (creatorOrcid.length > 0) {
+      s += "\t\t<nameIdentifier>"+creatorOrcid+"</nameIdentifier>\n"
+      s += "\t\t<nameIdentifierScheme>ORCID</nameIdentifierScheme>\n"
+    }
+    s += "\t</creator>\n"
+    s += "</creators>\n"
+
+    // Title
+    s += "<titles>\n\t<title>"+dataset.name+"</title>\n</titles>\n"
+
+    // Publisher
+    // TODO: Not sure Clowder is right here.
+    s += "<publisher>Clowder</publisher>\n"
+
+    // PublicationYear
+    val yyyy = new SimpleDateFormat("yyyy").format(dataset.created)
+    s += "<publicationYear>"+yyyy+"</publicationYear>\n"
+
+    // ResourceType
+    s += "<resourceType resourceTypeGeneral=\"Dataset\">Clowder Dataset</ResourceType>\n"
+
+    // ---------- RECOMMENDED/OPTIONAL FIELDS ----------
+
+    // Description
+    s += "<descriptions>\n\t<description descriptionType=\"Abstract\">"+dataset.description+"</description>\n</descriptions>\n"
+
+    // Contributors (anyone else who provided files or metadata)
+    val contribList = ListBuffer[String]()
+    datasets.getUserMetadataJSON(dataset.id).foreach(md => {
+      val mdAuth = md.toString
+      if (mdAuth.length > 1 && mdAuth != creatorName) contribList += mdAuth
+    })
+    files.get(dataset.files).found.foreach(fi => {
+      if (fi.author.fullName != creatorName) contribList += fi.author.fullName
+      files.getUserMetadataJSON(fi.id).foreach(md => {
+        val mdAuth = md.toString
+        if (mdAuth.length > 1 && mdAuth != creatorName) contribList += mdAuth
+      })
+    })
+
+    if (contribList.length > 0) {
+      s += "<contributors>\n"
+      contribList.distinct.foreach(name => s += "\t<contributor contributorType=\"\">"+name+"</contributor>\n")
+      s += "</contributors>\n"
+    }
+
+    // Date (Created)
+    val isoDate = new SimpleDateFormat("YYYY-MM-dd").format(dataset.created)
+    s += "<dates>\n\t<date dateType=\"Created\">"+isoDate+"</date>\n</dates>\n"
+
+    // AlternateIdentifier
+    s += "<alternateIdentifier>"+dataset.id.stringify+"</alternateIdentifier>\n"
+
+    // Format
+    s += "<format>application/zip</format>\n"
+
+    // Subjects
+    if (dataset.tags.length > 0) {
+      s += "<subjects>\n"
+      dataset.tags.foreach(t => s += "\t<tag>"+t.name+"</tag>\n")
+      s += "</subjects>\n"
+    }
+
+    // RelatedIdentifier
+    val clowder_url = "CLOWDER URL GOES HERE"
```

Comment:
What do you plan on doing for this url? Should we put the actual url before we merge this PR?

Reply:
I think we talked about it but I completely forgot! :(

Reply:
Now the original requests that call this enumerator will pass along the URL from their request so the dataset link will be included there.